### ISSUE TITLE
Latex code to import is not working for Xtabular

### ISSUE THREAD

**Issue by dibya-89** [JDMCreator/LaTeXTableEditor#100]
**Describe the bug**
Table can not import

**Answer the following questions**
1. Which browser are you using:

**JSON code**

Add the JSON code generated by the editor. To do so, click on "File > Generate JSON" at the top left corner of the editor. It will allow anyone to reproduce the table and any options you selected. Beware of any sensitive or confidential information in your table, as it will be part of the JSON code:

```
Your JSON code here.
```

** Related LaTeX code **

Insert any related LaTeX code here:
```
Your LaTeX code
\begin{xltabular}{\textwidth}{p{0.1\textwidth}p{0.18\textwidth}p{0.35\textwidth}p{0.25\textwidth}}

   \toprule
    \textbf{Author} & \textbf{Title} & \textbf{Description} &\textbf{Highlights}\\
    \midrule
    \endfirsthead
    
\midrule 
\multicolumn{4}{r}{{Continued on next page}} \\ 
\hline
\endfoot
\\

 \toprule
    \textbf{Author} & \textbf{Title} & \textbf{Description} &\textbf{Highlights}\\
\midrule



\endhead



\endlastfoot

    Ketan Joshi et al. \cite{joshi2020robust} & Robust sports image classification using inceptionV3 and neural networks&
     It presents a framework for sports image classification using Inception V3 and neural network and achieves an average accuracy of 96.64\% over six sports categories.
    
     A detailed comparison is also given with other classifiers such as Random Forest, K-Nearest Neighbors, etc., for effectiveness validation. 
    
    
    & In this work, sports images and videos are analyzed to develop  various applications such as blog writing, sports education, etc.  \\

    Russo et al. \cite{russo2019classification} &Classification of sports videos with combination of deep learning models and transfer learning&
    It proposes a deep learning-based approach that combines convolutional and recurrent neural networks to classify sports videos into 15 individual classes, achieving high test accuracy using transfer learning with the VGG-16 model. 

 &The proposed approach focuses on sports action-based classification by combining spatial and motion features extracted from CNN with temporal analysis using RNN. \\

 SSkandha et al. \cite{skandha2022novel} &A novel genetic algorithm-based approach for compression and acceleration of deep learning convolution neural network: an application in computer tomography lung cancer data & This work highlights the compression of deep neural network for ensuring its suitability towards IOT devices. Since lung cancer is one of a life-threatening diseases, it is essential to be detected using low-configuration devices at an early stage. 
 
 This work uses a Genetic Algorithm for model compression where the unwanted layers in the neural network are removed to improve the efficiency of the model. & The proposed approach reduces 90.3\% storage space and also improves the inference time by 35\%.\\
 
 Petrini et al. \cite{marino2023deep} &Deep neural networks compression: A comparative survey and choice recommendations &  The paper presents a comprehensive comparison of lossy and structure-preserving approaches to compress pre-trained convolutional neural networks (CNNs) and provides guidance for choosing the most suitable compression technique. 

The study includes experiments on two state-of-the-art CNNs and five benchmarks, analyzing the performance of compression techniques on both convolutional and fully-connected layers for classification and regression problems. & The experimental setting used to compare the compression techniques and described, including the use of two pre-trained CNN models and five datasets.
\\   
Simon Wiedemann et al.~\cite{simon}&DeepCABAC: A Universal Compression Algorithm 

for Deep Neural Networks &DeepCABAC is a compression algorithm for deep neural networks (DNNs) that applies Context-based Adaptive Binary Arithmetic Coder (CABAC) to the DNN parameters, achieving higher compression rates than previous techniques for DNN compression. 

It uses a novel quantization scheme that minimizes a rate-distortion function while considering the impact of quantization on DNN performance, allowing the representation of the entire network with just 9 MB.
& The algorithm is based on the H.264/AVC video coding standard and applies CABAC, which is a state-of-the-art lossless compression technique for video compression.

Experimental results show that DeepCABAC consistently achieves higher compression rates compared to previously proposed coding techniques for DNN compression.
\\
Podgorelec, V. et al.~\cite{Podgorelec}&Classification of similar sports images using a convolutional neural network with hyper-parameter optimization&The paper discusses the use of transfer learning in image classification, specifically for classifying sports images.

The paper discusses the use of transfer learning in image classification, specifically for classifying sports images. It presents a proposed image classification method and describes the conducted experiments and results. The authors also discuss the interpretation of the trained models using methods like LIME and SHAP. 
& The paper explores the use of transfer learning techniques, specifically fine-tuning, for image classification. 

Transfer learning involves training a model on a pre-trained model with adapted weight values, reducing training time and potentially improving predictive performance.\\
Gao, Y. et al.~\cite{gao2016improved}& Improved spatial pyramid matching for sports image classification
&
The paper addresses the need to consider both human pose and event scenes in sports image classification, using a combination of spatial pyramid matching (SPM) and Visual Words Spatial Dependence Matrices to improve classification accuracy. 

Experimental results show that the proposed method improves classification accuracy by approximately 19\% compared to SPM and outperforms other improved SPM methods in sports image classification. 
& 
The paper also mentions the use of the KSPM method, which focuses on improving the spatial position of objectives in sports images. It shows effective improvement in classification accuracy, particularly for sports scenes with athletes on a large scale.
\\
Huang, Pu.~\cite{huang2021sports}
&Sports Image Classification and Application Based on Visual Attention Analysis & The paper focuses on the classification and application of sports images using visual attention analysis, which simulates human eye recognition patterns and improves accuracy in classifying sports pictures. 

The study establishes a sports image classification system based on visual attention analysis and compares its effectiveness with other methods, showing significant advantages in terms of accuracy. & The results of the experiments show that the proposed method achieves an average accuracy of 34.5\%, which is significantly higher than the visual impairment method (8.5\%) and the core technology method (11.2\%). 
\\
Sarma, Moumita Sen, et al.~\cite{sarma2021traditional}&Traditional Bangladeshi sports video classification using deep learning method&The paper focuses on the classification of traditional Bangladeshi sports videos using deep learning techniques, specifically convolutional neural network (CNN) and long short term memory (LSTM) algorithms. A new dataset called Traditional Bangladeshi Sports Video (TBSV) is constructed, containing five classes of sports. The proposed model, which combines CNN and LSTM, outperforms previous works on challenging datasets and achieves an average accuracy of 99\% on the TBSV dataset. & The spatial features of sequential frames are extracted using CNN and then fed to an LSTM layer for analysis.
\\
Campr, Pavel, et al.~\cite{campr2014sports}&Sports video classification in continuous TV broadcasts&The paper focuses on classifying video footage or continuous TV broadcasts based on their content, using categories such as talk show, sport, movie, cartoon, and more specific topics like summer and winter Olympic sports. The classification is done by analyzing each frame of the video separately and then filtering the results in the time domain for more accurate and robust classification. The paper also discusses the selection of robust image features and classifiers, showing that complex features based on convolutional neural networks outperform simple feature extractors. &The paper compares several feature extraction methods and classifiers for image scene classification and topic classification in continuous videos. It applies these methods to standalone images as well as continuous videos without prior knowledge of topic changes. Cross-validation is used for more robust results, and the experiments are repeated with three random splits of the data.
\\
Farhad, Mohammad Yasir, et al. ~\cite{farhad2020sports}&Sports-net18: Various sports classification using transfer learning&The paper proposes a VGG16 transfer learning model to classify eighteen categories of various sports, achieving a promising result of 93\% accuracy. 

The authors have created their own sports dataset containing 9000 images and used deep learning techniques to accurately recognize and classify objects from sports images. &The proposed system consists of five convolutional blocks with different filter sizes and activation functions, followed by max-pooling and flattening layers. 
The authors have created their own sports dataset containing 9000 images for training and evaluation purposes.
\\
Song, H.~\cite{song2021secure}&Secure prediction and assessment of sports injuries using deep learning based convolutional neural network &The paper discusses the use of an optimized convolutional neural network (OCNN) based on deep learning to detect and assess sports injuries. It focuses on the extraction, study, and accuracy of complex algorithms for analyzing sports medical data. The OCNN model includes two convolutional layers, two pool layers, a fully connected layer, and a SoftMax structure for classification. The paper also proposes a cloud-based loop model for creating an advanced medical data network for sports medicine. Experimental results show that this approach provides technical support and guidance for deploying a specific cloud-based fusion system. &  The OCNN algorithm is used for data processing in the in-loop fusion simulation model, where the collected data is passed through the control layer and sent to the stored data center for processing. 

The paper suggests the use of a self-coding convolution neural network (SC neural network) that incorporates the configuration of the neural network of self-coding convolution to process and analyze multidimensional data.\\
 \bottomrule



\caption{Literature Review Summary}   
\label{tab:RL}
\end{xltabular}
